ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-incubation.yaml

```diff
@@ -47,7 +46,11 @@ tests:
     workflow: opendatahub-io-ci-image-mirror
 - as: opendatahub-operator-image-mirror
   postsubmit: true
+<<<<<<< HEAD
```

Comment:
You are committing conflict marks.

Reply:
yes, resolving it. My system kind of hand with thousands of files while syncing with upstream. I've resolved but due to system hang it commit.